### DISCUSSION TITLE
报个BUG

### DISCUSSION THREAD

**Discussion by webzl-zl** [WPFDevelopersOrg/WPFDevelopers#35] [General]
CropAvatar类的InitialImage方法，如果选择图片宽高相等，报错。只考虑了宽高不相等的情况，没有对宽高相等的情况赋值，导致image的宽和高都是NaN

**Comment by yanjinhuagood** [comment]
感谢反馈,有时间修复下。

**Comment by yanjinhuagood** [comment]
已修复  WPFDevelopers 1.1.0.1-preview8
